css/index.css

```diff
@@ -0,0 +1,64 @@
+.wrapper{
+    margin:0 auto;
+    width: 1000px;
+}
+body {
+    background-color: rgba(255, 228, 196, 0.281);
+    font-family:'Poppins',sans-serif;
+    border: 2px solid brown;
+    border-radius: 5px;
+}
+h1 {
+    color: chocolate;
+    font-size: 50px;
+    font-weight: 60;
+    text-align: center;
+    font-family:cursive;
+    
+}
+h2 {
+    color: chocolate;
+    font-size: 35px;
+    font-weight: 200;
+    font-family: fantasy;
+}
+p {
+    color: rgba(160, 81, 45, 0.925);
+    font-size: 20px;
+    font-family: monospace;
+    
+
+}
+a {
+    color: rgba(160, 81, 45, 0.925);
+
+    
+}
+
+section{
+    text-align: center;
+    margin: 10px 20px 30px 40px
+
+}
+h1,h2,p{
+    background-color: blanchedalmond;
+}
+ul {
+    list-style-type: none;
+    margin:0;
+    padding:0;
+
+}
+li {
+    display: inline;
+    color: rgba(160, 81, 45, 0.925);
+    
+}
+a {
+    display: block;
+    padding: 8px;
+    background-color:whitesmoke;
+}
+img{
+    border-radius:  8%;
+}
```

Comment:
for the anchor tags, it can be styled to display different states e.g visited, hover, active

.connect > a:link {
  color: red;
}

/* visited link */
a:visited {
  color: green;
}

/* mouse over link */
a:hover {
  color: hotpink;
}

/* selected link */
a:active {
  color: blue;
}